scripts/cubic/bootstrap-fmriprep.sh

```diff
@@ -92,60 +97,45 @@ else
     datalad save -r -m "added input data"
 fi
 
-SUBJECTS=$(find inputs/data -type d -name 'sub-*' | cut -d '/' -f 3 )
+# dj: using subset for testing
+SUBJECTS=$(find inputs/data -type d -name 'sub-NDARZZ*' | cut -d '/' -f 3 )
 if [ -z "${SUBJECTS}" ]
 then
     echo "No subjects found in input data"
     # exit 1
 fi
 
-set +u
-CONTAINERDS=$2
-set -u
-#if [[ ! -z "${CONTAINERDS}" ]]; then
-cd ${PROJECTROOT}
-datalad clone ${CONTAINERDS} pennlinc-containers
-## Add the containers as a subdataset
-#datalad clone ria+ssh://sciget.pmacs.upenn.edu:/project/bbl_projects/containers#~pennlinc-containers pennlinc-containers
-# download the image so we don't ddos pmacs
-cd pennlinc-containers
-datalad get -r .
-# get rid of the references to pmacs
-set +e
-datalad siblings remove -s pmacs-ria-storage
-datalad siblings remove -s origin
-set -e
-
-cd ${PROJECTROOT}/analysis
-datalad install -d . --source ${PROJECTROOT}/pennlinc-containers
+#dj: should we change this part?
+CONTAINERDS=///repronim/containers
+datalad install -d . --source ${CONTAINERDS}
+datalad get containers/images/bids/bids-fmriprep--20.2.3.sing
 
 ## the actual compute job specification
 cat > code/participant_job.sh << "EOT"
 #!/bin/bash
-#$ -S /bin/bash
-#$ -l h_vmem=25G
-#$ -l tmpfree=200G
-#$ -R y 
-#$ -l h_rt=24:00:00
+
 # Set up the correct conda environment
-source ${CONDA_PREFIX}/bin/activate base
-echo I\'m in $PWD using `which python`
 
+module load openmind/singularity/3.5.0
+echo SINGULARITY `singularity --help`
+source ${CONDA_PREFIX}/bin/activate datalad
+
+echo I\'m in $PWD using `which python`
 # fail whenever something is fishy, use -x to get verbose logfiles
 set -e -u -x
-
 # Set up the remotes and get the subject id from the call
+args=($@)
 dssource="$1"
 pushgitremote="$2"
 subid="$3"
-
+job_id="$4"
+echo SUBID: $subid
 # change into the cluster-assigned temp directory. Not done by default in SGE
-cd ${CBICA_TMPDIR}
+cd ${TMPDIR}
 # OR Run it on a shared network drive
 # cd /cbica/comp_space/$(basename $HOME)
-
 # Used for the branch names and the temp dir
-BRANCH="job-${JOB_ID}-${subid}"
+BRANCH="job-${job_id}-${subid}"
```

Comment:
i think there is a slurm variable here that can be used without passing it job id as an argument. you may want to echo env to see.